docs/changelog.txt

```diff
@@ -1,7 +1,41 @@
 [[changelog]]
 [appendix]
-== Release Notes
+Release Notes
```

Comment:
Now that #218 has been merged, please update this PR from the `0.1` branch and let me know when it's ready to re-review.